controller/DonateController.js

```diff
@@ -0,0 +1,57 @@
+const { Donate } = require("../services");
+const catchAsyncError = require("../middleware/catchAsyncError");
+
+/**
+ * @description Donate
+ * @route POST /v1/donate
+ * @access PUBLIC
+ */
+exports.donateNow = catchAsyncError(async (req, res) => {
+  /**
+   * @todo Handle validation for req.body
+   */
+
+  const donation = await Donate.makeDonation(req.body);
+  if (donation.success === true)
+    return res.status(201).json({
+      status: true,
+      message: "Follow this link to complete donation",
+      data: donation.data
+    });
+  else
+    return res.status(500).json({
```

Comment:
These are the areas we want to add logs to help us debug when an error happens.
Even if you're not doing it in this PR, please add some TODO comments so we can come back to it.